contents/product-engineers/pmf-surveys.md

```diff
@@ -0,0 +1,166 @@
+
+- experienced the core value-prop of your product.
+- used your product at least twice.
+- used your product in the last two weeks.
+
+You can begin analyzing the results once you have at least 30 responses. 
+
+### Step 2: Study your best customers
+
+Start by examining the users who would be "very disappointed" if they could no longer use your product. There are two things you need to learn about them:
+
+1. Who they are
+
+2. Why they love your product
+
+To find the answers, look at their responses to the questions `What is the main benefit you receive from our product?` and `What type of people do you think would most benefit from it?`.
+
+Then, use this information to create a [user persona](/product-engineers/how-to-create-user-personas) of your best supporters. Users who match this persona are the ones you need to improve your app for and you should focus your attention solely on them. In the next step we explain how to do this.
+
+![Personas of survey responders](../images/blog/pmf-surveys/personas.png)
+<Caption>Focus your attention only on users that match your user persona</Caption>
+
+### Step 3: Find your "on-the-fence" users
+
+Direct your attention to the users who responded with "somewhat disappointed". In particular, focus on the ones that:
+
+- match the user persona you created in the previous step, and
+- gave similar responses like your best supporters when asked about the main benefit they receive from your app.
+
+It's clear that these users are seeing the value in your app, but something is holding them back from becoming passionate users. Usually, only one or two things are in their way, so these users will be the easiest to convert into fanatics who champion your product.
+
+To find out what their problems are, look at their response to the question `"How can we improve our app for you?" and make a note of the common themes.
+
+### Step 4: Reprioritize your roadmap
+
+Now that you understand your app's most important benefits and missing features, the next step is to reprioritize your roadmap. 
+
+You need to continue improving the app's main benefits while also addressing the problems your users mentioned. Rahul Vohra, founder of Superhuman, explains why this is important:
+
+<BorderWrapper>
+<Quote
+    imageSource=""
+    size= "md"
+    name= "Rahul Vohra"
+    title= "Founder & CEO, Superhuman"
+    quote={`"If you only double down on what users love, your product-market fit score won't increase. If you only address what holds users back, your competition will likely overtake you."`}
+/>
+</BorderWrapper>
+
+When prioritizing your roadmap, you don't need to overthink things. Stack-ranking your projects by assigning each one a label of `low/medium/high` for effort and impact is sufficient (or you can even [add dollar estimates](/product-engineers/fixing-growth-problems#problem-2-prioritization)). This will give you a clear idea of which projects to work on first.
```

Comment:
How do you evaluate impact?

I am working on a related newsletter, I said "At a lower level, individual teams also have an idea of what all the potential features they can build are. This is formed based on their principles, user feedback, metrics,  experience, and opinions."